torchdynamo/variables/builder.py

```diff
@@ -273,19 +273,7 @@ def _wrap(self, value):
             return AutogradFunctionVariable(
                 value, guards=make_guards(GuardBuilder.FUNCTION_MATCH)
             )
-        if istype(
-            value,
-            (
-                np.int8,
-                np.int16,
-                np.int32,
-                np.int64,
-                np.uint8,
-                np.uint16,
-                np.uint32,
-                np.uint64,
-            ),
-        ):
+        elif is_numpy_int_type(value):
```

Comment:
On the `main` branch, there is no `elif`, but I think that was a typo.

Pointing it out to ensure review does not miss this.